### ISSUE TITLE
input a black pictrue, how to slove?

### ISSUE THREAD

**Issue by nobodykernel** [tencent-ailab/IP-Adapter#276]
code was  used as follow:
import torch
from diffusers import StableDiffusionPipeline, StableDiffusionImg2ImgPipeline, StableDiffusionInpaintPipelineLegacy, DDIMScheduler, AutoencoderKL
from PIL import Image

from ip_adapter import IPAdapter
base_model_path = "runwayml/stable-diffusion-v1-5"
vae_model_path = "stabilityai/sd-vae-ft-mse"
image_encoder_path = "models/image_encoder/"
ip_ckpt = "models/ip-adapter_sd15.bin"
device = "cuda"

from PIL import Image
def image_grid(imgs, rows, cols):
    assert len(imgs) == rows * cols

    w, h = imgs[0].size
    grid = Image.new('RGB', size=(cols * w, rows * h))
    grid_w, grid_h = grid.size

    for i, img in enumerate(imgs):
        grid.paste(img, box=(i % cols * w, i // cols * h))
    return grid
noise_scheduler = DDIMScheduler(
    num_train_timesteps=1000,
    beta_start=0.00085,
    beta_end=0.012,
    beta_schedule="scaled_linear",
    clip_sample=False,
    set_alpha_to_one=False,
    steps_offset=1,
)
vae = AutoencoderKL.from_pretrained(vae_model_path).to(dtype=torch.float16)

# load SD pipeline
pipe = StableDiffusionPipeline.from_pretrained(
    base_model_path,
    torch_dtype=torch.float16,
    scheduler=noise_scheduler,
    vae=vae,
    feature_extractor=None,
    safety_checker=None
)
# read image prompt
image = Image.open("assets/images/woman.png")
image.resize((256, 256))
# load ip-adapter
ip_model = IPAdapter(pipe, image_encoder_path, ip_ckpt, device)

# generate image variations
images = ip_model.generate(pil_image=image, num_samples=4, num_inference_steps=50, seed=42)
grid = image_grid(images, 1, 4)
grid.show()

cosole was as gived as fellow.
/home/un/.conda/envs/pt20gpu/bin/python /home/un/PycharmProjects/IP-Adapter/main.py 
Loading pipeline components...: 100%|██████████| 5/5 [00:00<00:00, 13.18it/s]
You have disabled the safety checker for <class 'diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion.StableDiffusionPipeline'> by passing `safety_checker=None`. Ensure that you abide to the conditions of the Stable Diffusion license and do not expose unfiltered results in services or applications open to the public. Both the diffusers team and Hugging Face strongly recommend to keep the safety filter enabled in all public facing circumstances, disabling it only for use-cases that involve analyzing network behavior or auditing its results. For more information, please have a look at https://github.com/huggingface/diffusers/pull/254 .
100%|██████████| 50/50 [00:07<00:00,  6.66it/s]
/home/un/.conda/envs/pt20gpu/lib/python3.8/site-packages/diffusers/image_processor.py:88: RuntimeWarning: invalid value encountered in cast
  images = (images * 255).round().astype("uint8")

Process finished with exit code 0

but,the result is a black picture/
![tmpptv4mj16](https://github.com/tencent-ailab/IP-Adapter/assets/157854986/49255919-2d18-4985-8f0a-8af9ffc4a5dd)

**Comment by masaisai111** [comment]
I wonder if you solved it
